backend/main.py

```
@@ -0,0 +1,213 @@
+# Endpoints
+@app.get("/")
+def read_root():
+    return {"message": "Welcome to Numyp API! Go to /docs to see Swagger UI."}
+
+# Auth
+@app.post("/auth/signup")
+def signup(user: schemas.UserCreate):
+    return {"message": f"User {user.username} created successfully"}
+
+@app.post("/auth/login", response_model=schemas.Token)
+def login(form_data: OAuth2PasswordRequestForm = Depends()):
```

Ruff (0.14.7)

Unused function argument: `form_data`

(ARG001)